crates/codegen/syntax_templates/src/typescript/parser_output.rs

```diff
@@ -32,15 +34,20 @@ impl ParseOutput {
 #[napi]
 #[derive(PartialEq, Clone)]
 pub struct ParseError {
-    pub(crate) position: usize,
+    pub(crate) position: InputPosition,
```

Comment:
Performance of the error getter? I might be missing something.
This will only be called once for each error, only when there are errors. Is there somewhere else it can impact?

Reply:
I wanted it to match the node position api i.e. separate char_ and byte_. I did that because it will be a common operation and otherwise it involves a lot of work to create the object across the FFI boundary.

Maybe I'm not getting your point.

Reply:
Not blocking this PR. Let's discuss f2f in that case :)